config.example.ts

```diff
@@ -38,15 +40,21 @@ export const EMAIL = {
 	]
 };
 
-export const CHANNELS = {			// Channel IDs
+export const CHANNELS = { // Channel IDs
 	ERROR_LOG: '',
 	SERVER_LOG: '',
 	MEMBER_LOG: '',
 	MOD_LOG: '',
 	FEEDBACK: '',
 	SAGE: '',
 	ANNOUNCEMENTS: '',
-	ARCHIVE: ''
+	ARCHIVE: '',
+	ROLE_SELECT: ''
+};
+
+export const ROLE_DROPDOWNS = {
+	COURSE_ROLES: '', // course roles
```

Comment:
Lol, comments saying exactly what the variables are named

Reply:
listennnnn it was for my brain